<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeWindow</class>
 <widget class="QMainWindow" name="EncodeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSB Steganography - Encode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #282a36;
}

QLabel {
    color: #f8f8f2;
    font-size: 14px;
    margin-left: 10px;
    margin-right: 10px;
    margin-top: 10px;
    margin-bottom: 5px;
}

QLineEdit {
    background-color: #44475a;
    color: #f8f8f2;
    border: 1px solid #6272a4;
    border-radius: 5px;
    padding: 5px;
    margin-left: 10px;
    margin-right: 10px;
    margin-top: 5px;
    margin-bottom: 10px;
    font-size: 14px;
}

QPushButton {
    background-color: #6272a4;
    color: #f8f8f2;
    border: none;
    border-radius: 5px;
    padding: 10px;
    margin-left: 10px;
    margin-right: 10px;
    margin-top: 5px;
    margin-bottom: 10px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #8be9fd;
    color: #282a36;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LSB Steganography - Encode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblInputImage">
      <property name="text">
       <string>Input Image:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txtInputImage"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectInput">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblFileToHide">
      <property name="text">
       <string>File to Hide:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtFileToHide"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectFile">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnEncode">
      <property name="text">
       <string>Encode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblEncodedImagePath">
      <property name="text">
       <string>Encoded Image Path:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEncodedImagePath">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnViewEncodedImage">
      <property name="text">
       <string>View Encoded Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>